addons/product_expiry/models/stock_move.py

```diff
@@ -5,7 +5,7 @@
 import dateutil.parser as dparser
 from re import findall as re_findall
 
-from odoo import fields, models
+from odoo import fields, models, api
```

Comment:
[import ordered](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#imports)
```suggestion
from odoo import api, fields, models
```